bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/caller/Caller.java

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.core.caller;
+
+import java.io.Closeable;
+import java.util.concurrent.CompletionStage;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Definition of a simple caller.
+ *
+ * @author Markus Rathgeb - initial contribution
```

Comment:
Okay for me, I assume I see that line in lower case in other source files, too.
Will search all files for the lower case variant the next days and open a PR, that change all occurences of "initial contribution" to "Initial contribution".

It would be nice if SAT will complain about a lower case "i" if we want to start with an upper case.

Reply:
AFAIK we have the [AuthorContributionDescriptionCheck](https://github.com/openhab/static-code-analysis/blob/8cc0af5e53a0d53f82837b2cdc165e7261bb4892/sat-plugin/src/main/resources/rulesets/checkstyle/rules.xml#L229-L233) to check for it but the implementation uses `containsIgnoreCase` to validate the contribution description.

Reply:
If we agree to use correct upper and lower case, ignore case does not make sense.
Should a issue created on SAT?